packages/app-project/src/components/ProjectHeader/ProjectHeader.spec.js

```diff
@@ -1,62 +1,173 @@
+import nock from 'nock'
 
-import { ProjectHeader } from './ProjectHeader'
-import Avatar from './components/Avatar'
-import ProjectTitle from './components/ProjectTitle'
-import ApprovedIcon from './components/ApprovedIcon'
-import UnderReviewLabel from './components/UnderReviewLabel'
-import LocaleSwitcher from './components/LocaleSwitcher'
-
-const TITLE = 'Project title'
+import * as Stories from './ProjectHeader.stories.js'
+import ProjectHeader from './ProjectHeader.js'
 
 describe('Component > ProjectHeader', function () {
-  let wrapper
+  let locale, navMenu, pageURL, projectAvatar, projectBackground, projectTitle
 
   before(function () {
-    wrapper = shallow(<ProjectHeader title={TITLE} />)
+    nock('https://talk-staging.zooniverse.org')
+    .persist()
+    .get('/notifications')
+    .query(true)
+    .reply(200, {})
+    .get('/conversations')
+    .query(true)
+    .reply(200, {})
   })
 
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
+  after(function () {
+    nock.cleanAll()
   })
 
-  it('should render a `<Avatar />` component', function () {
-    expect(wrapper.find(Avatar)).to.have.lengthOf(1)
-  })
+  describe('default behaviour', function () {
+    before(function () {
+      render(<Stories.NotLoggedIn {...Stories.NotLoggedIn.args} />)
+      projectAvatar = screen.getByRole('img', { name: 'ProjectHeader.Avatar.alt' })
+      projectBackground = document.querySelector(`div[aria-hidden] div[data-src='https://panoptes-uploads.zooniverse.org/project_background/7a3c6210-f97d-4f40-9ab4-8da30772ee01.jpeg']`)
+      projectTitle = screen.getByRole('heading', { level: 1, name: 'Snapshot Serengeti' })
+      navMenu = screen.getByRole('navigation', { name: 'ProjectHeader.ProjectNav.ariaLabel' })
+    })
 
-  it('should render a `<ProjectTitle />` component', function () {
-    expect(wrapper.find(ProjectTitle)).to.have.lengthOf(1)
-  })
+    it('should display the project title', function () {
+      expect(projectTitle).to.be.ok()
+    })
+
+    it('should show the project avatar', function () {
+      expect(projectAvatar).to.exist()
+    })
 
-  it('should render a `<ApprovedIcon />` component', function () {
-    expect(wrapper.find(ApprovedIcon)).to.have.lengthOf(1)
+    it('should show the project background', function () {
+      expect(projectBackground).to.exist()
+    })
   })
 
-  it('should not render a `<UnderReviewLabel />` component', function () {
-    expect(wrapper.find(UnderReviewLabel)).to.have.lengthOf(0)
+  describe('when not logged in', function () {
+    let navLinks
+
+    before(function () {
+      const { NotLoggedIn } = Stories
+      render(<NotLoggedIn {...NotLoggedIn.args} />)
+      projectAvatar = screen.getByRole('img', { name: 'ProjectHeader.Avatar.alt' })
+      projectTitle = screen.getByRole('heading', { level: 1, name: 'Snapshot Serengeti' })
+      const navMenu = screen.getByRole('navigation', { name: 'ProjectHeader.ProjectNav.ariaLabel' })
```

Comment:
This label is missing from the project navigation on small screens, but those aren’t tested here.